apps/summon-app/src/utils/content.ts

```diff
@@ -2,51 +2,51 @@ export const FORM_COPY = {
+    description: 'Configure some advanced governance settings.',
   },
   SHAMANS: {
     title: 'Starting Shamans',
     description:
-      'Shamans are very powerful as they can have administrative control over voting and non-voting stakes. Be very careful adding shamans.',
+      'Shamans are very powerful as they can have administrative control over key components of the DAO, including voting and non-voting stake. Be very careful adding shamans.',
   },
   MEMBERS: {
     title: 'Starting Members',
     description:
-      'You must have at least one member to start. Add other starting members as desired. You can always add more members later through a proposal or a shaman.',
+      'You must have at least one member to start. Add other starting members as desired. You can always add more members later through a proposal.',
   },
 };
 export const INFO_COPY = {
   VOTING_STK:
-    "Voting stake tokens are created when the DAO is summoned. What does your community want to name the DAO's token?",
+    "The voting stake token name is set when the DAO is summoned. What does your community want to name the DAO's token?",
   TOKEN_SYMBOL:
-    'This is the common shorthand referring to stake among DAO members.  What abbreviated symbol would your DAO like to use here?',
+    'This is the common shorthand your DAO will use to refer to voting stake among DAO members. What abbreviated symbol would your DAO like to use here?',
   STAKE_TRANSFER:
-    'Allow or disallow the voting stake tokens to be transferable between addresses.',
+    'Should DAO members be allowed to transfer their voting stake to other accounts? After summoning, the DAO can change this with a proposal.',
   NV_STAKE_TRANSFER:
-    'Allow or disallow the non-voting stake tokens to be transferable between addresses',
+    'Should DAO members be allowed to transfer their non-voting stake to other accounts. After summoning, the DAO can change this with a proposal.',
   VOTING_PERIOD:
-    'The length of time a proposal will remain available for voting stake members to submit their vote',
+    'The length of time a proposal will remain open for voting stake members to submit votes.',
   GRACE_PERIOD:
-    'The time period between a proposal’s approval and its execution.  This provides time for members to ragequit their shares, before the approved proposal is executed.',
+    'The time period between a proposal’s approval and its execution. This provides time for members to ragequit their shares, before the approved proposal is executed.',
   QUORUM:
-    'The minimum percentage of DAO members needed to approve a proposal for it to execute.  This percentage can greatly affect DAO operations',
+    'The minimum percentage of DAO members (voting stake) needed to approve a proposal for it to be executed. This percentage can greatly affect DAO operations, though the DAO can change it with a proposal after summoning with a proposal.',
```

Comment:
Would make "the DAO can change it with a proposal after summoning with a proposal" its own sentence.